src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -111,13 +111,59 @@ const acdcChangeHandler = async (
 const AppWrapper = (props: { children: ReactNode }) => {
   const dispatch = useAppDispatch();
   const authentication = useAppSelector(getAuthentication);
-  const [agentInitErr, setAgentInitErr] = useState(false);
+  const [isOnline, setIsOnline] = useState(false);
```

Comment:
I tested this out locally and we've a problem here: when KERIA goes offline, nothing updates this to false. We only set it to online when it comes back online.

This means the queue is always unpaused for incoming requests if we are logged in.